package exception;

public class room {
	public static void main[(String[] args){
		try {
		int a=10;
		int b=0;
		int c=a/b;
		int arr[]=new int[5];
		System.out.println(arr[6]);
		System.out.println(c);
		}catch(ArithemticException e) {
			System.out.println("Error");
		}
		finally {
			System.out.println("End of program");
		}
	}

}

